### DISCUSSION TITLE
Some various ideas

### DISCUSSION THREAD

**Discussion by JeanRoxus** [janstehno/cotw-companion#34] [Ideas]
## - Reservations/Maps
1. When you enter the “reservations/maps” section you have the option to save some of them as favorites.
2. Favorite “reservations/maps” appear on the main screen when you open the application for faster use of the maps you use most frequently.

## - Animals
3. When entering the map, when looking at the top where it says the percentage of animal appearances, it is also a pressable button and gives you quick and summarized access to the “need areas” such as the times of: food, feeding and rest.
4. If you search for an animal from the reservations/maps section, make sure that the "needs zone" schedule that appears is from the map from which you initially searched for it.
5. In the image that shows the anatomy of the animal, have the possibility of increasing or decreasing the "opacity" of the skin to have a reference of where to take the shot. Ex: above the elbow, at the end of the skull, in the middle of the thorax, etc.
6. Indicate which is the trophy organ of each animal to prevent that shot.

## - Weapons
7. A recommended set for each map that covers the classes of the animals it contains
8. Weapons that belong to any DLC, indicate which DLC they belong to.
9. Indicate when a weapon, ammunition, consumable or claim has a requirement such as rifle, bow, pistol or shotgun points. Also if your purchase requirement is a minimum level.

## - Others
10. Add a section that allows you to enter the level and "plan" how to increase points in skills and advantages.
(This idea is not my own, I saw it on this page and I liked it).
https://thehunter.dpenet.net/
11. After selecting a map, before seeing the fauna and its categories, some icons appear at the top. I feel like they should give a lot of information about the map and they also look good, but I'm not really sure what each one means. I think it might be a good idea that when you press or hold it it tells you what it means.

### I tried to show what my ideas will be like to allow me to understand better, I hope the ideas are within what is possible, I hope you like some of them and can include them in the application.

Each photograph is saved with the name of the point to which it refers.

![1](https://github.com/janstehno/cotwcompanion/assets/81821598/0e8a3031-ccba-491e-9115-dae4377cb65e)
![2](https://github.com/janstehno/cotwcompanion/assets/81821598/0ba9310e-8d54-4999-a5cb-c6b30fb2285f)
![3 1](https://github.com/janstehno/cotwcompanion/assets/81821598/b9c3e810-e82f-4826-9de9-47dd7e0d903f)
![3 2](https://github.com/janstehno/cotwcompanion/assets/81821598/899337d0-c9e8-48f3-a798-03dcde5ffc93)
![10](https://github.com/janstehno/cotwcompanion/assets/81821598/4b21b4a8-6035-479c-9e4a-790d2223acf7)
![11](https://github.com/janstehno/cotwcompanion/assets/81821598/5382b2ae-74ef-4447-a9b9-ab2803429dd2)

**Comment by janstehno** [comment]
Hello, and thank you for your suggestions, Also thank you for the screenshots. This is a rarity. Almost no one cares about the part where I have to think about how the suggestion/feature will look like and what is the best way to approach it for users to not hate it. Now I will try to answer all of them and what I think about its importance and implementation.

Reservations
---
1. `Easy` Favourite reservations are possible feature. Like the idea. Could be implemented for more things probably.
2. `Easy` Not sure about this, I would have to find some different location for them. On small screens there is already lack of area on the home screen and I would like to keep it as simple as possible.

Animals
---
3. `Mediocore` This is a nice idea, however same as the home screen, the area is just too small to add there something like need zones. The text would be too small in my opinion. I will try to think about it.
5. ✅ `Mediocore` If you are talking about the suggestion above, then this would be a no-brainer. No need to point it out. If you are talking about the need zones in the animal's information card, then it is possible to show the need zones from the reserve you showed the animal's information card from. But I do not think it is that hard to just swipe it to the right need zone circle. I will think about it.
6. `Easy` Opacity of skin is possible feature. I already planned adding bones and making the opacity system for them instead.
7. `Mediocore` I do not think this is something important (same goes for just poiting out by text what is the trophy organ). When animal has antlers, horns, or tusks, don't shoot the head. For me this is a pointless information to show.

Weapons
---
8. `Mediocore` This is fairly possible. I would probably have to use the same method I am using for each animal's best recommended weapons.
9. ✅ `Easy` Showing what DLC weapons and callers belong to is not impossible. Probably would be shown in the detailed information card.
10. ✅ `Mediocore` Unlock requirments are really great idea. It would take a lot of time to collect the values probably. Don't know how to graphically show it at the moment. I will think about it.

Other
---
11. ✅ `Hard` If this is something I would not have to save for later use, then it is fine. Otherwise I am not really sure about this.
12. ✅ `Easy` Yeah, the icons are just random idea. It just shows what environment you can expect in the reserve. Snowflake for winter, trees for forest, etc. I did not think it is hard to understand them. Maybe I could add some title to it, something like "Environment".

I hope this gives you my point of view. We can for sure work it out. I would also probably split this into more suggestions here on GitHub later on as this is fairly complex as "one" idea.

**Comment by janstehno** [comment]
Just so you know, I have closed and split this discussion. The ones I have created are more specific and contain only 1 or 2 points from this discussion. All of them have link back to this discussion so all can see the original suggestions.
